### ISSUE TITLE
report lake update outputs

### ISSUE THREAD

**Issue by Seasawher** [oliver-butterley/lean-update#33]
when update fails, the action should report `lake update` outputs.

I think `lake build --log-level=error` is useful for this

**Comment by oliver-butterley** [comment]
Yes, would be good to have clear and detailed report on failure.
